usagov/vote-gov-drupal
# VOTE-1894/95 - Add NVRF Page content type and NVRF Field vocabulary

## Jira ticket

[VOTE-1894](https://bixal-projects.atlassian.net/browse/VOTE-1894) - Add NVRF Page content type
[VOTE-1895](https://bixal-projects.atlassian.net/browse/VOTE-1895) - Add NVRF Field vocabulary

## Description

Insert a brief summary of the changes included in this PR and any additional information or context which may help the reviewer.

## Deployment and testing

### Pre-deploy

1. none

### Post-deploy

1. `drush cim` should be executed to enable the new content type and vocabulary.
2. `lando drush cim' - for local environment

### QA/Test

1. Login as an admin
3. Goto /admin/structure/types/manage/nvrf_page
4. Confirm that the Content fields available meet the requirements according to the following screenshot:
<img width="1142" alt="Screen Shot 2023-08-15 at 11 59 01 AM" src="https://github.com/usagov/vote-gov-drupal/assets/60625545/8cd67e69-a609-484c-a04f-5aea8af7c3fc">

5. Goto /admin/structure/taxonomy/manage/nvrf_field/overview/fields
6. Confirm that the NVRF Fields vocabulary fields meets the requirements according to the following screenshot:
<img width="825" alt="Screen Shot 2023-08-15 at 11 57 47 AM" src="https://github.com/usagov/vote-gov-drupal/assets/60625545/6757b64f-13b7-4280-9bdb-1a63bd09b8ba">

## Checklist for the Developer




## Checklist for the Peer Reviewers
